doc/decisions/meta-decisions/use-arc42-for-architecture-overview-documents/index.md

```diff
@@ -0,0 +1,62 @@
+# Adoption of Architecture Overview Documents
+
+**Status**: Proposed
```

Comment:
Missing the `!!! info` formatting here to render these in the template's info banner.

e.g. 
```markdown
!!! info

    **Status**: Proposed 

    **Level**: ...
```